### DISCUSSION TITLE
I'm mining and I don't see anything in my wallet

### DISCUSSION THREAD

**Discussion by brianp** [tari-project/tari#6263] [Q&A] [answered]
I've been mining all day, or a week and I don't see anything in my wallet. What's wrong?

**Comment by brianp** [comment, marked as answer]
When you mine you are not guaranteed to win a block. There are many factors involved while mining that can affect your chances.

Here's the best way to *estimate* what your chances might be to win a block:

- Get your hashrate for either xmrig or sha3 mining
- Get the network hashrate from [explore.tari.com](https://explore.tari.com)
- Divide your hashrate by the network hashrate, multiply by 100. This will give you the percentage of blocks you should win. (This is just an estimate, you are not guaranteed to win these)
- There are around 350 blocks generated per day for xmrig and 350 for sha3, so multiply that percentage by 350 to work out how many blocks per day you can win. If the number is less than 1, then it will take multiple days to win a block

**Reply by stringhandler** [reply]
For example, if you have 1% of the RandomX network hashrate, you should win around 3.5 blocks per day. Some days you might get 2 blocks, some days 0, some days 5 blocks.

**Comment by stringhandler** [comment]
It's also worth seeing this discussion: https://github.com/tari-project/tari/discussions/6332
